src/app/cli/src/coda.ml

```diff
@@ -19,6 +19,18 @@ let maybe_sleep _ = Deferred.unit
 
 [%%endif]
 
+let chain_id =
```

Comment:
Seems like you don't need a `lazy` value if you only `force` it once, though it doesn't really hurt anything.

Reply:
It's more about where the forcing happens than how many times it happens, yeah? I guess I should  have inlined the computation into the record construction instead.